src/core/style/Colors.js

```diff
@@ -0,0 +1,3 @@
+export default {
+  BLUE: '#046B99'
```

Comment:
I see you're using LUK blue colors in other places, so what is this for? Perhaps rename to like `CA_BLUE`?